misc/mouse.talon

```diff
@@ -47,8 +49,12 @@ midclick:
 	mouse_click()
 	# close the mouse grid
 	user.grid_close()
-drag: 
-	user.mouse_drag()
+drag:
+	user.mouse_drag(0)
+	# close the mouse grid
+	user.grid_close()
+right drag:
```

Comment:
This is pretty nitpicky, but I'd vote to stay with "righty drag" for consistency